.github/workflows/release.yml

```diff
@@ -39,7 +40,7 @@ jobs:
         run: |
           export PATH=$PATH:$HOME/.foundry/bin
           cd contracts/
-          forge build
+          forge build --force --extra-output-files bin
```

Comment:
why is `--force` and `--extra-output-files` needed now?

Reply:
`--force` isn't strictly needed, but I think good to add if for whatever reason output files from previous builds were around on the machine the workflow is run on (this should never be the case within Github Actions itself).

I added `--extra-output-files bin` so that the bytecode file (`.bin`) is generated to be uploaded with the release artifacts.